server/src/modules/jobs/effectifs/import-effectifs-mail.ts

```diff
@@ -0,0 +1,14 @@
+import { getTdbRupturant } from "../../../common/apis/tdb";
+import { insertBrevoContact, processBrevoContact } from "../../actions/tdb.actions";
+
+export async function hydrateEffectifsEmail() {
+  const data = await getTdbRupturant();
+
+  for (const item of data) {
+    await insertBrevoContact(item);
+  }
```

Comment:
C'est assez lent de faire comme ça, tu peux utiliser un `bulkWrite` plutot ? 

```suggestion
  await getDbCollection("brevo.contacts").bulkWrite(data.map(getUpsertBrevoContactOp), { unordered: true );
```